server/views/partials/new_workAndSkillsPage/goals/index.njk

```diff
@@ -4,10 +4,18 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {%- call summaryCard({title: "Goals", id: "goals"}) -%}
 
-    {% include './vc2GoalsOnly.njk' %}
-    {# {% include './plpGoalsOnly.njk' %} #}
-    {# {% include './plpAndVc2Goals.njk' %} #}
-    {# {% include './noGoals.njk' %} #}
-    {# {% include './problemRetrievingGoals.njk' %} #} 
+    {% if problemRetrievingPrisonerGoalData %}
+        {% include './problemRetrievingGoals.njk' %}
+    {% else %}
+        {% if hasVc2Goals and not hasPlpGoals %}
+            {% include './vc2GoalsOnly.njk' %}
+        {% elif hasPlpGoals and not hasVc2Goals %}
+            {% include './plpGoalsOnly.njk' %}
+        {% elif hasPlpGoals and hasVc2Goals %}
```

Comment:
You `could`  use the property `hasVC2GoalsAndPlpGoals` that you setup here ... but actually I think I prefer the readability of it here with the `and` (it kinda mirrors the lines above that use `and not`; which I like)

So ... up to you - could could either use your property `hasVC2GoalsAndPlpGoals` here , or remove it altogether from the controller code above on the basis that you're not using it